sphinx/source/docs/user_guide/styling.rst

```diff
@@ -20,6 +20,29 @@ following ways:
 
 .. _userguide_styling_visual_properties:
 
+Using Palettes
+-----------------
+
+Palettes are sequences (lists or tuples) of RGB(A) hex strings that define a
+colormap and can set as the ``palette`` attribute of all chart types from
+``bokeh.charts`` and as the ``color`` attribute of many plot objects from
+``bokeh.plotting``. Bokeh offers many of the standard Brewer palettes, which
+can be imported from the ``bokeh.palettes`` module. For example, importing
+“Spectral6” gives a linearly-spaced, six element list of RBG(A) hex strings
```

Comment:
"linearly-spaced" refers to the action the color-mapper, which linearly maps scalar values between a fixed range into the color bins. I would just say "provides a six element list..."